src/main/java/com/exasol/report/ReportWriter.java

```diff
@@ -0,0 +1,98 @@
+package com.exasol.report;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import com.exasol.UserInput;
+
+/**
+ * This class writes reports from {@link ValidationReport} to a file.
+ */
+public class ReportWriter {
+    private static final Logger LOGGER = Logger.getLogger(ReportWriter.class.getName());
+    private static final String LINE_SEPARATOR = System.getProperty("line.separator");
```

Comment:
use constant from `ReleaseRobotConstants` ?

Reply:
Forgot to remove it from here